### PR TITLE
added function new meeting for admin

### PR DESCRIPTION
**Pull request by GoralTomaszGorski**
used th:case expression to present clear free and booked therms

### CODE REVIEW

File: src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java [hunk 1]
```diff
@@ -26,5 +43,31 @@ public String meetingAdmin(Model model){
...
+        return "admin/meeting-admin-form";
+
+    }
 
+    @PostMapping("/admin/dadaj-rezerwacje")
+    public String addMeeting(MeetingVisitorSaveDto meetingSave, RedirectAttributes redirectAttributes) {
+        meetingService.addMeeting(meetingSave);
+        redirectAttributes.addFlashAttribute(
+                AdminController.NOTIFICATION_ATTRIBUTE,
+                "Pomyłśnie dokonano zapisu dla %s na %s"
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java]: fix typo "Pomyłśnie"

Comment (gorskimarcin96) [reply src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java]: i would change it `"Pomyłśnie dokonano zapisu dla %s na %s"` to `"Pomyślnie dokonano zapisu dla <b>%s</b> na <b>%s</b>"`

then we have user friendly notification:
image

@GoralTomaszGorski what do you think?

and remember, you need change `th:text` to `th:utext` when you are using html tag in string
`<p class="notification-bar" th:if="${notification}" th:utext="${notification}">`

**Review by gorskimarcin96 — COMMENTED** [review]
we can write unit test for sending form or not yet?

File: src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java (hunk 1 above)
Comment (GoralTomaszGorski) [reply src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java]: thx for your help

Comment (gorskimarcin96) [reply src/main/java/goral/psychotherapistoffice/web/admin/MeetingManagementController.java]: When you fixed code for my comment then you can click "Resolve conversation" and finally when you are resolved all comments then you can click "Merge pull request".

File: src/main/java/goral/psychotherapistoffice/web/visitor/VisitorController.java [hunk 2]
```diff
@@ -37,41 +39,55 @@ public VisitorController(PatientService patientService, CalenderService calender
...
     }
 
-    @PostMapping("/visitor/dadaj-rezerwacje")
-    public String addMeeting(MeetingVisitorSaveDto meetingSave, RedirectAttributes redirectAttributes) {
-        meetingService.addMeeting(meetingSave);
-
+    @PostMapping("/termin/{id}")
+    public String addPatientAndMeeting(@ModelAttribute("patientDto") PatientDto patientDto, @ModelAttribute("meetingSave") MeetingVisitorSaveDto meetingSave, RedirectAttributes redirectAttributes){
+        patientService.addPatient(patientDto);
+        meetingSave.setPatient(patientDto.getId());
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/web/visitor/VisitorController.java:70]: I am thinking about this... we have in this exception, am i right?

Comment (gorskimarcin96) [reply src/main/java/goral/psychotherapistoffice/web/visitor/VisitorController.java:70]:
```java
    @Transactional
    public void addPatient(PatientDto patientDto){
        Patient patientToSave = new Patient();
        patientToSave.setNick(patientDto.getNick());
        patientToSave.setName(patientDto.getName());
        patientToSave.setSurname(patientDto.getSurname());
        patientToSave.setTelephone(patientDto.getTelephone());
        patientToSave.setYearOfBrith(patientDto.getYearOfBrith());
        patientRepository.save(patientToSave);

    }
```
so, when we run this method `patientRepository.save(patientToSave);` maybe entity has id value. in my symfony technology when we save entity in database then automatically it is getting id value

you can try this
```java
    @Transactional
    public PatientDto addPatient(PatientDto patientDto){
        Patient patientToSave = new Patient();
        patientToSave.setNick(patientDto.getNick());
        patientToSave.setName(patientDto.getName());
        patientToSave.setSurname(patientDto.getSurname());
        patientToSave.setTelephone(patientDto.getTelephone());
        patientToSave.setYearOfBrith(patientDto.getYearOfBrith());
        patientRepository.save(patientToSave);

        return patientToSave;
    }
```
    
and in controller
    
you can try replace this
```java
patientService.addPatient(patientDto);
meetingSave.setPatient(patientDto.getId());
```

for this code

```java
meetingSave.setPatient(patientService.addPatient(patientDto).getId());
```

**Pull request merged**